workflow-core/api/workflow-core.api

```diff
@@ -1,10 +1,43 @@
 public abstract interface class com/squareup/workflow1/BaseRenderContext {
```

Comment:
Can we make this an abstract class? Would save us all the synthetic methods, and all the concrete impls in both concrete subclasses.

Reply:
Tried, failed. The version with no event type confuses the compiler if they're extensions. Same in Kotlin 1.4. We suspect a compiler bug, will file.

Reply:
https://youtrack.jetbrains.com/issue/KT-42741